README.md

```diff
@@ -102,6 +102,20 @@ $ adb shell am broadcast -a io.appium.settings.location -n io.appium.settings/.r
 
 The first value in the returned `data` string is the current latitude, the second is the longitude and the last one is the altitude. An empty string is returned if the data cannot be retrieved (more details on the failure cause can be found in the logcat output).
 
+## Setting Mock Locations
+
+Start sending scheduled updates (every 2s) for mock location with the specified values by executing:
+```shell
+$ adb shell am startservice --user 0 -n io.appium.settings/.LocationService --es longitude {longitude-value} --es latitude {latitude-value}
```

Comment:
can we also support altitude here?

Reply:
In Oreo and higher (26+), sometimes i get  "*Error: app is in background *" while using
`adb shell am startservice`
can we update it to
`adb shell am start-foreground-service`?